src/FulfillerMultiSelect.jsx

```diff
@@ -119,20 +148,53 @@ class FulfillerMultiSelect extends React.Component {
     async getRecentFulfillerIds() {
         if (this.props.accessToken) {
             let settings = await this.customizrClient.getSettings(this.props.accessToken);
-            let recentFulfillerIds = settings.recentFulfillerIds || [];
-            this.setState({ recentFulfillerIds });
-            return recentFulfillerIds;
+            return {
+                recentFulfillerIds: settings.recentFulfillerIds || [],
+                recentFulfillerIdsMulti: settings.recentFulfillerIdsMulti || []
+            };
+        }
+    }
+
+    recentFulfillersChanged(fulfillerIds) {
+        const recentFulfillerIds = this.state.recentFulfillerIds;
+
+        if (recentFulfillerIds.length === 0) {
```

Comment:
I think you could even have `(fulfillerIds.length <= recentFulfillerIds.length)`